### DISCUSSION TITLE
[Feature] Show slideshow with real transitions (without a black intermediate image)

### DISCUSSION THREAD

**Discussion by MakerMeik** [immich-app/immich#14968] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I have started using Immich's slideshow function for a “digital picture frame” consisting of a Raspberry Pi and the corresponding 7” display. I am basically very satisfied with the solution. But if you are not consciously looking at the display, the image transitions are very unpleasant due to the black image that can be seen between the pictures. Especially if you only have the display in your peripheral vision. 
Unfortunately, the “Show slideshow transition” option does not help either, as a short black image is also visible here. I would therefore be pleased if there were a proper option in future to fade the images into one another.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by mmomjian** [comment]
There are two great slideshow solutions, Immich Kiosk and Immich Frame. I recommend using one of those. 

Duplicate: #5095

**Reply by MakerMeik** [reply]
Oh, that had slipped past me. Thank you very much, I have set up Immich-Kiosk and it is running perfectly.
